### PR TITLE
Adding network-perf step across our configs

### PR DESCRIPTION
**Pull request by jtaleric**
Adding network-perf

### CODE REVIEW

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-stable-prod-perfscale-rosa-singleaz-network-perf-2nodes	periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-singleaz-network-perf-2nodes	periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-perfscale-rosa-singleaz-network-perf-2nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests-net-perf

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly.yaml [hunk 1]
```diff
@@ -250,6 +250,20 @@ tests:
     test:
     - ref: openshift-qe-ingress-perf
     workflow: rosa-aws-sts
+- as: perfscale-rosa-singleaz-network-perf-2nodes
```
Comment (svetsa-rh) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly.yaml:253]: Same here. "periodic" at end of test name.

**Review by svetsa-rh — CHANGES_REQUESTED** [review]

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-singleaz-network-perf-2nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-perfscale-rosa-singleaz-network-perf-2nodes

**Comment by svetsa-rh** [comment]
@jtaleric Can we add hcp tests too?

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-stable-prod-perfscale-rosa-singleaz-network-perf-2nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-stable-prod-perfscale-rosa-hcp-1zone-network-perf-2nodes-periodic periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-perfscale-rosa-singleaz-network-perf-2nodes-periodic periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests-net-perf periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-singleaz-network-perf-2nodes-periodic periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-hcp-1zone-network-perf-2nodes-periodic

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-stable-prod-perfscale-rosa-singleaz-network-perf-2nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-hcp-1zone-network-perf-2nodes-periodic periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-perfscale-rosa-singleaz-network-perf-2nodes-periodic periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-stable-prod-perfscale-rosa-hcp-1zone-network-perf-2nodes-periodic

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-hcp-1zone-network-perf-2nodes-periodic pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-stable-prod-perfscale-rosa-singleaz-network-perf-2nodes

**Comment by svetsa-rh** [comment]
@jtaleric I see that https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/41453/rehearse-41453-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-hcp-1zone-network-perf-2nodes-periodic/1682066376350502912 actually ran but failed at the end while deleting the stacks. So, we can ignore this failure for now.

Asking someone to help take a look here https://redhat-internal.slack.com/archives/CH76YSYSC/p1689885119584019?thread_ts=1670328469.221149&cid=CH76YSYSC

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-stable-prod-perfscale-rosa-singleaz-network-perf-2nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-stable-prod.yaml [hunk 2]
```diff
@@ -68,6 +68,38 @@ tests:
...
+      MULTI_AZ: "false"
+      OPENSHIFT_VERSION: "4.12"
+      REGION: us-west-2
+      REPLICAS: "2"
+      ROSA_LOGIN_ENV: production
+    test:
+    - chain: openshift-qe-network-perf
+    workflow: rosa-aws-sts
+- as: perfscale-rosa-hcp-1zone-network-perf-2nodes-periodic
+  cron: 0 7 * * 1
```
Comment (svetsa-rh) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-stable-prod.yaml:87]: @jtaleric Do we want to run this in production on a cron basis for 4.12? Just want to make sure that it was not added here as periodic job accidentally.

**Review by svetsa-rh — COMMENTED** [review]
@jtaleric Pls see my comment regarding 4.12 prod network HCP test.

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-stable-prod-perfscale-rosa-singleaz-network-perf-2nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-stable-prod-perfscale-rosa-singleaz-network-perf-2nodes

**Pull request closed without merging**